CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## v2.0.30 - 2021-01-26
```

Comment:
Since we release on every other Wednesday, please update to the 27th. Also line breaks to match the other entries.